src/redux/motorcycles/motorcycles.js

```diff
@@ -0,0 +1,51 @@
+export const FETCH_MOTORCYCLES = 'BOOK-APPOINTMENT/MOTORCYCLES/FETCH_MOTORCYCLES';
+export const CREATE_MOTORCYCLE = 'BOOK-APPOINTMENT/MOTORCYCLES/CREATE_MOTORCYCLE';
+export const DELETE_MOTORCYCLE = 'BOOK-APPOINTMENT/MOTORCYCLES/DELETE_MOTORCYCLE';
+export const FETCH_SINGLE_MOTORCYCLE = 'BOOK-APPOINTMENT/MOTORCYCLES/FETCH_SINGLE_MOTORCYCLE';
+export const UPDATE_MOTOR = 'BOOK-APPOINTMENT/MOTORCYCLES/UPDATE_MOTOR';
+
+export const fetchMotorcycles = (payload) => ({
+  type: FETCH_MOTORCYCLES,
+  payload,
+});
+
```

Comment:
Great job so far. But there is no call to the API in this function. You can refactor the functions like the example below.

```
export const fetchMotorcycles = () => (dispatch)=> {
  API.fetchMotors((response) => {
    dispatch({
       type: FETCH_MOTORCYCLES,
       payload: response.data,
     });
  });
};
```